src/pharmpy/model/execution_steps.py

```diff
@@ -272,6 +264,29 @@ def _canonicalize_and_check_method(method: str) -> str:
             )
         return method
 
+    @staticmethod
+    def _canonicalize_derivatives(derivatives: tuple) -> tuple:
+        if not isinstance(derivatives, tuple):
```

Comment:
This is not what we want. For `create` any `Sequence` should be fine. We simply convert it to `tuple`.

Reply:
Oh ok, i misunderstood this then, will change it to allow any sequence